src/Controls/src/Core/Compatibility/Handlers/Shell/iOS/ShellItemRenderer.cs

```diff
@@ -113,9 +113,9 @@ public override void ViewDidLoad()
 
 			ShouldSelectViewController = (tabController, viewController) =>
 			{
-				bool accept = true;
+				bool accept = false;
 				var r = RendererForViewController(viewController);
-				if (r != null)
+				if (r is not null && r != SelectedViewController)
```

Comment:
Could include an UITest?

Reply:
I have added the test case. Could you please review it and let me know if you have any concerns?